odd-platform-api/src/main/resources/db/migration/V0_0_87__create_relation_tables.sql

```diff
@@ -0,0 +1,36 @@
+CREATE TABLE IF NOT EXISTS relationship
+(
+    id                         bigserial PRIMARY KEY,
+    name                       varchar(512),
+    relationship_oddrn         varchar(512) UNIQUE,
+    source_dataset_oddrn       varchar(512)       NOT NULL,
+    target_dataset_oddrn       varchar(512)       NOT NULL,
+    is_manualy_created         boolean,
+    relationship_status        SMALLINT,
+    last_ingested_at           TIMESTAMP WITHOUT TIME ZONE,
+    deleted_at                 TIMESTAMP WITHOUT TIME ZONE
+);
+
+CREATE TABLE IF NOT EXISTS erd_relationship
```

Comment:
I don't mind but it feels like `erd_relationship_details` is more semantically correct. Current naming pattern feels like `erd_relatatioships` table **inherits** `relationships` table which is not true

Reply:
done